mkosi.md

```diff
@@ -544,12 +544,15 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
 
 `SkeletonTree=`, `--skeleton-tree=`
 
-: Takes a path to a directory to copy into the OS tree before invoking
-  the package manager. Use this to insert files and directories into
-  the OS tree before the package manager installs any packages. If
-  this option is not used, but the `mkosi.skeleton/` directory is
-  found in the local directory it is automatically used for this
-  purpose (also see the "Files" section below).
+: Takes a colon separated pair of paths. The first path refers to a
+  directory to copy into the OS tree before invoking the package
+  manager. The second path refers to the target directory inside the
+  image. If the second path is not provided, the directory is copied
+  on top of the root directory of the image. Use this to insert files
+  and directories into the OS tree before the package manager installs
+  any packages. If this option is not used, but the `mkosi.skeleton/`
+  directory is found in the local directory it is automatically used
+  for this purpose (also see the "Files" section below).
```

Comment:
Let's maybe be super clear about `mkosi.skeleton/` doesn't support a target and uses the root directory
```suggestion
  for this purpose with the root directory as target (also see the "Files" section below).
```
same below for `ExtraTree=`